include/irr/asset/IMeshPacker.h

```diff
@@ -78,26 +93,33 @@ class IMeshPacker
         if (m_vtxSize)
         {
             m_vtxBuffAlctrResSpc = _IRR_ALIGNED_MALLOC(core::GeneralpurposeAddressAllocator<uint32_t>::reserved_size(alignof(std::max_align_t), allocParams.vertexBuffSupportedSizeInBytes / m_vtxSize, allocParams.vertexBufferMinAllocSize /*divided by vtxSize?*/), _IRR_SIMD_ALIGNMENT);
-            //TODO: check if m_alctrReservedSpace[n] != nullptr
+            //for now mesh packer will not allow mesh buffers without any per vertex attributes
+            _IRR_DEBUG_BREAK_IF(m_vtxBuffAlctrResSpc == nullptr);
+            assert(m_vtxBuffAlctrResSpc != nullptr);
```

Comment:
`m_vtxSize==0` if you have no per-vertex attributes, so you'll never hit this assert ;)

Reply:
but it is possible to use valid mesh buffer with per instance attributes only..

Reply:
but you check for presence of per-vertex attributes in the `if` statement just above